sympy/utilities/lambdify.py

```diff
@@ -202,75 +208,253 @@ def lambdify(args, expr, modules=None, printer=None, use_imps=True,
     ``x`` and SymPy functions ``sin`` and ``cos``, then converted it to an
     equivalent NumPy function ``f``, and called it on a NumPy array ``a``.
 
-    .. warning::
-       This function uses ``exec``, and thus shouldn't be used on unsanitized
-       input.
+    Parameters
+    ==========
 
-    Arguments
-    =========
+    args : List[Symbol]
```

Comment:
Do you mean `[List | Symbol]`?

Reply:
I don't think it was easy to find right annotation for list. But is it right?
I see people are using so many different ways to annotate the types. 
But I think that there should be a convention that can be corresponding to the `typing`, because I see napoleon builder generates automatic reference for annotated types, such that users can navigate to the doc to see what `Symbol` is.